### ISSUE TITLE
Not ready to be used

### ISSUE THREAD

**Issue by oleg-alexandrov** [rafapages/multitex#5]
I liked the paper so I thought I would give it a try. I spent a lot of time trying to make it work, including reading the code, compiling the code and debugging things, and trying with various dense meshes either produced by other SfM tools or robot depth point clouds. I also studied them in meshlab, and I tried to clean them in various ways. 

It crashes in many places when it does various mess processing tasks and also in "block packing". There are no even basic kind of sanity checks when trying to access indices from arrays or when dereferencing iterators. 

Add to it that the fact that there is not even a simple example or dataset for how it works, or what expectations are placed on input meshes.

In short, this package needs a lot of work to make it usable though though the underlying idea is sound. I am giving up.

**Comment by rafapages** [comment]
Hi @oleg-alexandrov, for some weird reason all the github notifications were going to spam and I just saw your comments. I'm sorry you invested your time and you decided to give up. I uploaded this repo years ago and I haven't done any proper maintenance almost since I left academia.

I still would like it to be useful, so I will try to find some time to give some sample data or to separate the mesh unwrapping from the proper texturing, even if it is too late for you!

Sorry again!

**Comment by zhixiongzuo** [comment]
As said above ,there really are many problems for making it work, could you give a sampe data for your work ? Thanks.

**Comment by oleg-alexandrov** [comment]
@rafapages , I understand how things work. There is always more to do than there is time.

@zhixiongzuo , one solution I used is texrecon (https://github.com/nmoehrle/mvs-texturing) I would not be able to say how it compares to SSMVTex, but only that it was easy to set up and did what I needed it to.

**Comment by zhixiongzuo** [comment]
@oleg-alexandrov , It 's kind of you , I will have a try . Thank you very much !

**Comment by marcomiglionico94** [comment]
> @rafapages , I understand how things work. There is always more to do than there is time.
> 
> @zhixiongzuo , one solution I used is texrecon (https://github.com/nmoehrle/mvs-texturing) I would not be able to say how it compares to SSMVTex, but only that it was easy to set up and did what I needed it to.

Hello man, I am trying to use mvs-texturing and I am having some issues. Can i ask you some questions about it? Thanks a lot

**Comment by oleg-alexandrov** [comment]
Sure. I am not its developer but worked enough with mvs-texturing for big projects and it did the job. You can write to me at oleg.alexandrov@gmail.com. 

As before, no offence intended for the authors of SSMVtex. It takes a lot of funding and engineering work to handle all possible situations in a software package.
